db/sql/003-change-tenant-id-to-uuid-in-time-entry-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE timecard.time_entry
+ALTER COLUMN owner_id TYPE VARCHAR(36);
```

Comment:
Research seems to suggest that using postgress' native uuid type is more efficient that using varchar(36). 
I know the other uuid types have been stored as varchar but perhaps this is the point to start using the native type..

Reply:
as discussed separately, we'll do this change along with the other existing ones in a separate ticket